treeherder/model/models.py

```diff
@@ -1074,7 +1075,8 @@ class FailureLine(models.Model):
     status = models.CharField(max_length=7, choices=STATUS_CHOICES)
     expected = models.CharField(max_length=7, choices=STATUS_CHOICES, blank=True, null=True)
     message = models.TextField(blank=True, null=True)
-    signature = models.TextField(blank=True, null=True)  # Prefix index length 50
+    # Prefix index: signature(50)
+    signature = models.TextField(blank=True, null=True, db_index=True)
```

Comment:
I would be surprised if postgres didn't have a limit on the size of a field for an index as well?

Now that you mention it, I think a more detailed comment would be helpful for the index_together case below as well. If an index-declaration has no functional effect, we should just take it out and replace it with a comment IMO. Otherwise someone's going to be looking at this file a year or two later and wonder what "db_index" or the "index_together" statements *really* do (which is nothing).